### ISSUE TITLE
It will be worst in -1 minutes

### ISSUE THREAD

**Issue by farski** [jeffbyrnes/hubot-forecast#3]
There's some situation where this happens:

```
WEATHER: 100% chance of inclement weather in the next hour for at least 39 minutes. It will be worst in -1 minutes (light precipitation). 
```

Seems...wrong.

**Comment by jeffbyrnes** [comment]
@farski ew, yeah. Haven’t seen that in the wild but… yeah.

On a separate note, I’m definitely granting you collab rights to this repo; this is your code, after all! Would love to set you up to push to the NPM package as well.

**Comment by martzmakes** [comment]
+1 on this issue, just saw it.  WEATHER: 39% chance of inclement weather in the next hour for at least 61 minutes. It will be worst in 7 minutes (very light precipitation). http://forecast.io/#/f/38.906400,-77.038158/1434678477

[9:52 PM] officebotBOT:
Weather Update!
WEATHER: The weather should be clear for at least an hour.

[9:57 PM] officebotBOT:
Weather Update!
WEATHER: 44% chance of inclement weather in the next hour for at least 61 minutes. It will be worst in -1 minutes (very light precipitation). http://forecast.io/#/f/38.906400,-77.038158/1434679077

**Comment by jeffbyrnes** [comment]
Yeah, I’ve seen it a few times as well. Haven’t had a chance to investigate thoroughly yet.

**Comment by EdHurtig** [comment]
This seems like the culprit: https://github.com/jeffbyrnes/hubot-forecast/blob/master/src/forecast.coffee#L97-L101

**Comment by jeffbyrnes** [comment]
Partly solved by #14; that code will, at least, prevent the “-1 minutes” response (instead, it will be “0 minutes”).

I’ve got a larger refactor in the works that should address this.
